libnavigation-ui/src/main/java/com/mapbox/navigation/ui/alert/AlertView.java

```diff
@@ -124,23 +149,21 @@ private void initAnimations() {
     slideDownTop.setInterpolator(new OvershootInterpolator(2.0f));
   }
 
-  private void initBackground() {
-    if (android.os.Build.VERSION.SDK_INT < android.os.Build.VERSION_CODES.LOLLIPOP) {
-      int progressColor = ThemeSwitcher.retrieveThemeColor(getContext(),
-        R.attr.navigationViewProgress);
-      int progressBackgroundColor = ThemeSwitcher.retrieveThemeColor(getContext(),
-        R.attr.navigationViewProgressBackground);
-
-      LayerDrawable progressBarDrawable = (LayerDrawable) alertProgressBar.getProgressDrawable();
-      // ProgressBar progress color
-      Drawable progressBackgroundDrawable = progressBarDrawable.getDrawable(0);
-      progressBackgroundDrawable.setColorFilter(progressBackgroundColor, PorterDuff.Mode.SRC_IN);
+  private void applyCustomizedAttributes() {
+    CardView alertCardView = findViewById(R.id.alertCardView);
+    alertCardView.setCardBackgroundColor(backgroundColor);
+    alertText.setTextColor(textColor);
 
+    LayerDrawable progressBarDrawable = (LayerDrawable) alertProgressBar.getProgressDrawable();
+    // ProgressBar progress color
+    Drawable progressBackgroundDrawable = progressBarDrawable.getDrawable(0);
+    progressBackgroundDrawable.setColorFilter(progressBarBackgroundColor, PorterDuff.Mode.SRC_IN);
 
-      // ProgressBar background color
-      Drawable progressDrawable = progressBarDrawable.getDrawable(1);
-      progressDrawable.setColorFilter(progressColor, PorterDuff.Mode.SRC_IN);
+    // ProgressBar background color
+    Drawable progressDrawable = progressBarDrawable.getDrawable(1);
+    progressDrawable.setColorFilter(progressBarColor, PorterDuff.Mode.SRC_IN);
 
+    if (Build.VERSION.SDK_INT < Build.VERSION_CODES.LOLLIPOP) {
```

Comment:
Comparing with the change above the above mentioned logic is only executed for devices that are pre-lollipop. But in the new change implemented we do it always. I just want to be sure that it is desired

Reply:
yes, this is intentionally to keep same as before. while I tested this, 
```
getBackground().setAlpha(0);
``` doesn't work on OS >= 21. So I keep the original implementation here.